.circleci/config.yml

```diff
@@ -64,10 +89,8 @@ workflows:
   version: 2
   build_test_and_deploy:
     jobs:
-      - validate:
-          filters:
-            tags:
-              ignore: /.*/
+      - validate
+      - validate_and_publish_pact
```

Comment:
i think the name of this job is a bit confusing with the repetition of 'validate', it makes it sound like it does the same as 'validate' + some more stuff.

Reply:
21c0b1525959201bf3fc4c5e96ff801209badbc0 has `pact_check_and_publish`, what do you think?